src/core/endpoints/reel/WatchEndpoint.ts

```diff
@@ -0,0 +1,19 @@
+import type {IReelWatchRequest, ReelWatchEndpointOptions} from '../../../types/index.js';
+
+export const PATH = '/reel/reel_item_watch';
+
+/**
+ * Builds a `/reel/reel_watch_sequence` request payload.
+ * @param opts - The options to use.
+ * @returns The payload.
+ */
+export function build(opts: ReelWatchEndpointOptions): IReelWatchRequest {
+  return {
+    context: {},
```

Comment:
You don't need context here (or in the other files, that is added automatically with the session values, when YouTube.js does the request.

Reply:
Removed